public/stylesheets/characters.css

```diff
@@ -244,4 +244,62 @@ body.modal-open {
     overflow: auto;
     display: block; }
 
+#characters-container {
```

Comment:
Forgot to mention, but the project is using SCSS/SASS to compile the css files. You should run `npm run grunt` and edit the SCSS files instead. You can copy/paste the new CSS changes into the file if you want.

Reply:
Realized that. Most of my changes later were on the SCSS file. I'll revert the changes to this one, not sure how I ended up touching this one.

Reply:
Resolved in 4d233786654297f44f44304ca4710b92770a02a3

Regenerated the file.